tests/b2share_functional_tests/test_record_cli.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of EUDAT B2Share.
+# Copyright (C) 2017 SurfSara, University of Tübingen
+#
+# B2Share is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# B2Share is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with B2Share; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Test B2Share records cli module."""
+from __future__ import absolute_import, print_function
+
+import json
+from click.testing import CliRunner
+from flask_cli import ScriptInfo
+from flask import url_for
+
+from b2share.modules.communities.api import Community
+from b2share.modules.deposit.api import PublicationStates
+from b2share.modules.records.cli import check_dois
+from b2share_unit_tests.helpers import create_user, generate_record_data
+
+headers = [('Content-Type', 'application/json'),
+           ('Accept', 'application/json')]
+patch_headers = [('Content-Type', 'application/json-patch+json'),
+                 ('Accept', 'application/json')]
+
+def test_record_doi_cli(app, test_communities, login_user):
+    """Test adding a community and setting its schema using CLI commands."""
+    with app.app_context():
+        community_name = test_communities.popitem()[0]
+        community = Community.get(name=community_name)
+        com_admin = create_user('com_admin', roles=[community.admin_role])
```

Comment:
You can use [`test_community`](https://github.com/EUDAT-B2SHARE/b2share/blob/4d988f7a1b5d8678c234553f23685cc70b001363/tests/conftest.py#L205) fixture which creates the com_admin already.